Toolchain/install-chocolatly.ps1

```diff
@@ -0,0 +1,18 @@
+# Check if running as administrator
```

Comment:
is installing chocolatey packages as admin really required? This script seems a bit strange. Why not just point users to the upstream install instructions?